docs/tutorial/store-with-encryption.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Store with Encryption
+id: store-with-encryption
+---
+
+In the swarm, all data is public by default. To protect sensitive content, it must be encrypted so that only authorised users are able to decrypt and then view the plaintext content. 
+
+The Bee client provides a facility to encrypt files and directories while uploading which are only able to be read by users with access to the corresponding decryption key.
+
+# Encrypt and Upload a File
+
+To encrypt a file simply include the `Swarm-Encrypt: true` header with your `http` request.
+
+```sh
+$ curl -F file=@kitten.jpg -H "Swarm-Encrypt: true" http://localhost:8080/files
+```
+
+When successful, the Bee client will return a 64 byte reference, instead of the usual 32 bytes.
+
+```json
+{"reference":"9af0181735d4870589edc8748879dfd8898e0412eb63091c503212c4e23891823c484dcb8102baa7bd84117c5cc0176275efabaefccc9ac4a0e75ba3be9f1605"}
+```
+
+This consists of a 32 byte Swarm Reference, and a 32 byte decryption key.
+
+:::info
+Encryption is disabled by default on all Swarm Gateways to keep your data safe.
```

Comment:
Warn: never encrypt and decrypt your content on a computer that is not under your control (e.g. a public gateway). If you do so, it is not guaranteed that that your content will leak.

Reply:
(Can warn here, instead of line 37)